pom.xml

```diff
@@ -85,19 +85,35 @@
       <id>release</id>
       <build>
         <plugins>
+        <plugin>
```

Comment:
why do we need to add the config of maven-deploy-plugin?

Reply:
yes it is redundant as already while calling mvn deploy we are setting skiptest so this configuration is not required, I will update the changes